applications/DG-Max/DGMaxProgramUtils.cpp

```diff
@@ -148,4 +149,76 @@ PointPath<DIM> parsePath(const std::string& path) {
 template PointPath<1> parsePath(const std::string& path);
 template PointPath<2> parsePath(const std::string& path);
 template PointPath<3> parsePath(const std::string& path);
+
+std::unique_ptr<ZoneInfoStructureDefinition> readZonedDescription(
```

Comment:
Could you add a small example of such a file as a comment, please? :)

Reply:
Now included in the documentation of `determineStructureDescription`